MergerCli/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "GENERAL": {
     "validate": false,
+    "defaultConnectionLimit": 100,
```

Comment:
What kind of connection limit? For what? Depending on the answer, this may not be the correct place for this type of configuration.
Remove the 'default' prefix.
In addition, this configuration is not used.

Reply:
fixed to servicePointManagerDefaultConnectionLimit